src/main/java/mil/dds/anet/WaitForDBCommand.java

```diff
@@ -0,0 +1,76 @@
+package mil.dds.anet;
+
+import org.skife.jdbi.v2.DBI;
+
+import io.dropwizard.cli.ConfiguredCommand;
+import net.sourceforge.argparse4j.inf.Namespace;
+import mil.dds.anet.config.AnetConfiguration;
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.jdbi.DBIFactory;
+import io.dropwizard.setup.Environment;
+import net.sourceforge.argparse4j.inf.Subparser;
+
+
+public class WaitForDBCommand extends ConfiguredCommand<AnetConfiguration> {
+
+	public WaitForDBCommand() {
+		super("waitForDB","Waits until DB is ready for connection");
+	}
+
+	@Override
+	public void configure(Subparser subparser) {
+		subparser.addArgument("-n", "--nbAttempts")
+		.dest("dbConnectionNbAttempts")
+		.type(Integer.class)
+		.required(false)
+		.setDefault(20)
+		.help("Nb of attempts before giving up. 20 by default");
+
+		subparser.addArgument("-d", "--delay")
+		.dest("dbConnectionDelay")
+		.type(Integer.class)
+		.required(false)
+		.setDefault(500)
+		.help("Delay in ms between attempts. 500 by default");
```

Comment:
Indent all the lines starting with `.`.
I'd also make the defaults into `private static final int CONSTANT_X = yy;` and use those here.